sources/packages/backend/libs/utilities/src/esdc-utils.ts

```diff
@@ -1,27 +1,22 @@
-import { OfferingIntensity, RelationshipStatus } from "@sims/sims-db";
+import { RelationshipStatus, OfferingIntensity } from "@sims/sims-db";
 
 /**
- * Gets the marital status code conversion for the MSFAA request file
+ * Gets the offering intensity code conversion for the MSFAA request file
  */
-export function getMaritalStatusCode(
-  maritalStatus: RelationshipStatus,
-): string {
-  if (maritalStatus === RelationshipStatus.Married) {
-    return "M";
-  }
-  return maritalStatus === RelationshipStatus.Single ? "S" : "O";
+export function getOfferingIntensityCode(offeringIntensity: string): string {
+  return offeringIntensity === OfferingIntensity.fullTime ? "FT" : "PT";
 }
 
 /**
- * Gets the Part-Time marital status code conversion for the MSFAA request file.
+ * Gets the marital status code conversion for the MSFAA request file
  */
-export function getPartTimeMaritalStatusCode(
+export function getMaritalStatusCode(
   maritalStatus: RelationshipStatus,
 ): string {
   if (maritalStatus === RelationshipStatus.Married) {
-    return "MA";
+    return "M";
```

Comment:
Did these values change as a part of this story? `MA` -> `M` 
`SI` -> `S`
`SP` -> `O`

If so that can be added to story and PR description as well.